ophyd/controls/positioner.py

```diff
@@ -425,8 +421,6 @@ def __init__(self, setpoint, readback=None,
                                         name=self.name))
 
             self._readback.subscribe(self._pos_changed)
-
-            self._set_position(self._readback.value)
```

Comment:
So in the case that the read-back PV has yet to connect, `motor.position` will return `None`. You should probably do a connectivity check in the `position` property.